components/camel-salesforce/camel-salesforce-component/src/main/java/org/apache/camel/component/salesforce/SalesforceConsumer.java

```diff
@@ -290,6 +294,24 @@ protected void doStart() throws Exception {
         subscribed = true;
     }
 
+    /**
+     * If alsoStopSubscription is added, any underlying subscriptions will be stopped as well.
+     *
+     * SubscriptionHelper also logs out, so this will terminate the salesforce session as well.
+     *
+     * @param alsoStopSubscription
+     */
+    public void stop(boolean alsoStopSubscription) {
+        stop();
+        if (alsoStopSubscription) {
+            try {
+                subscriptionHelper.stop();
+            } catch (Exception e) {
+                LOG.warn("Failed to stop subscription due to: " + e.getMessage());
```

Comment:
This should be: 

```
                LOG.warn("Failed to stop subscription due to: {}", e.getMessage(), e);
```

Reply:
Thanks. Just a note: I meant without concatenating the log string at all.

Use log marker like this:

```
LOG.warn("Failed to stop subscription due to: {}", e.getMessage(), e);
```

Instead of concatenating the last part of the log message:
```
LOG.warn("Failed to stop subscription due to: " + e.getMessage(), e);
```

Reply:
Oh, of course. Sorry I overlooked that part. I have updated the code.